packages/zosuss/src/Shell.ts

```diff
@@ -93,8 +94,11 @@ export class Shell {
                                 stdoutHandler(dataToPrint);
                                 dataToPrint = "";
                                 isUserCommand = false;
-                            } else if (isUserCommand) {
-                                // print out the user command result
+                            }
+                            else if (dataToPrint.includes(command)) {
```

Comment:
I was able to reproduce the original issue by editing `~/.profile` to change my login shell from `sh` to `bash`.

The way this is currently written, if a user has `sh` as their login shell which does not return the command at the start of its output, we may end up stripping out text that we shouldn't be.
For example if a user runs `zowe uss issue cmd "pwd"` and their home directory is `/z/pwdaniels`, the "pwd" would be stripped from the output resulting in `/z/aniels`.

Should we make this check more strict? Perhaps something like the following:
`if (dataToPrint.trimStart().startsWith("$ " + command))`